src/zope/formlib/interfaces.py

```diff
@@ -870,14 +870,16 @@ class IFormField(Interface):
     )
 
     prefix = schema.ASCII(
-        constraint=reConstraint('[a-zA-Z][a-zA-Z0-9_]*',
-                                "Must be an identifier"),
+        constraint=reConstraint(
+            r'[a-zA-Z][a-zA-Z0-9_\.]*', "Must be an identifier or empty",
```

Comment:
Minor: you don't need to backslash-escape the period inside `[` ... `]`.

Reply:
TIL